courses/free-courses-en.md

```diff
@@ -1551,6 +1558,7 @@
 * [MOOC Introduction to Cyber Security 2023](https://cybersecuritybase.mooc.fi) - University of Helsinki
 * [Nightmare](https://guyinatuxedo.github.io/index.html) - guyinatuxedo
 * [Professor Messer’s SY0-601 CompTIA Security+ Course](https://www.professormesser.com/security-plus/sy0-601/sy0-601-video/sy0-601-comptia-security-plus-course/) - Professor Messer
+* [Python for Hackers FULL Course | Bug Bounty & Ethical Hacking](https://www.youtube.com/playlist?list=PLBfoYdk-WAlzfRFHRKAmA7Vw2svoGqfb5) - PhD Security
```

Comment:
I think this would fit better under the python heading.

Reply:
Hello @Thenlie, I wanted to let you know that I've carefully considered your suggestion and implemented the necessary adjustments. 
Regards

Reply:
Thanks for making these changes! 🚀